source/common/ssl/connection_impl.cc

```diff
@@ -15,6 +15,20 @@
 namespace Envoy {
 namespace Ssl {
 
+namespace {
+// TODO(mattklein123): Currently we don't populate local address for client connections. Nothing
+// looks at this currently, but we may want to populate this later for logging purposes.
+Network::Address::InstanceConstSharedPtr
+getNullLocalAddress(const Network::Address::Instance& address) {
```

Comment:
In general I prefer to not duplicate code, but don't care that much. It's fine to leave it like this if you want.